packages/component/src/Utils/addTargetBlankToHyperlinksMarkdown.spec.js

```diff
@@ -1,7 +1,12 @@
+// TODO: [P4] Object.fromEntries is not on Node.js 11.*
```

Comment:
file issue

Reply:
Will remove this line and move to CONTRIBUTING.md.

This is actually not a TODO on the code, but a TODO on every of us to make sure we have installed Node.js >= 12 before running tests. Some test code is using new ES features in Node.js 12.